sipa/model/misc.py

```diff
@@ -7,7 +7,7 @@
 
 TransactionTuple = namedtuple('Transaction', ['datum', 'value'])
 
-PaymentDetails = namedtuple('PaymentDetails', 'recipient bank iban bic purpose')
+PaymentDetails = namedtuple('PaymentDetails', 'recipient iban purpose')
```

Comment:
if you're changing this anyway, it would be more idiomatic to use a modern typed namedtuple:
```python
import typing as t  # at beginning of file
class PaymentDetails(t.NamedTuple):
    recipient: str
    iban: IBAN
    purpose: str
```
This way, the linter catches bugs when this tuple is mis-used.